### PR TITLE
docs: replaced dead s3 docs url and added attributes table

### PR DESCRIPTION
**Pull request by tk-nguyen**
**What type of PR is this?**
documentation

**Which issue does this PR fix**:
Broken S3 storage driver docs URL.

**What does this PR do / Why do we need it**:
- Included an attribute table
- Fix the broken URL.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by rchincha** [comment]
@tk-nguyen pls add the words in the SpellCheck ci failure here:
https://github.com/project-zot/project-zot.github.io/blob/main/.wordlist.txt
(alphabetically sorted)

**Comment by rchincha** [comment]
Once you are done, pls also sign your commit.
https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

File: docs/articles/storage.md [hunk 1]
```diff
@@ -243,8 +243,34 @@ To configure an Amazon Simple Storage Service (s3) bucket for zot, use the `stor
     }
 ```
 
-For descriptions of the configurable attributes for `storageDriver`, see the [s3 storage driver](https://github.com/docker/docker.github.io/blob/master/registry/storage-drivers/s3.md) project in GitHub.
-
+The following table lists the attributes of `s3` storage driver:
+
```
Comment (mbshields) [inline docs/articles/storage.md:247]: In the code example above this change, `storage` contains an attribute called `storageDriver` and that attribute contains a `name` attribute, which in this example has a value of `s3`.   Did this schema change?  

When you refer to the "attributes of `s3` storage driver," do you mean "attributes of `storageDriver` when configuring s3 storage"?  If so, should `name` be in the table, or is s3 the only type of storage driver supported?

Comment (tk-nguyen) [reply docs/articles/storage.md:247]: Yes, I meant "attributes of `storageDriver` when configuring s3 storage". According to https://zotregistry.dev/v2.0.1/articles/storage/?h=storage#remote-filesystem, only s3 storage driver is supported.

**Comment by tk-nguyen** [comment]
> @tk-nguyen pls add the words in the SpellCheck ci failure here: https://github.com/project-zot/project-zot.github.io/blob/main/.wordlist.txt (alphabetically sorted)

> Once you are done, pls also sign your commit. https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

Done.

File: docs/articles/storage.md (hunk 1 above)
Comment (mbshields) [reply docs/articles/storage.md:247]: You should change line 246 to clarify what you mean:
"The following table lists the configurable attributes of `storageDriver` when configuring s3 storage."

By the way, is `name` no longer a configurable attributes of `storageDriver`?  If not, it should be removed from the code example, but if it still exists, it should be added to the table.

Comment (tk-nguyen) [reply docs/articles/storage.md:247]: Will do.

**Review by mbshields — APPROVED** [review]
lgtm

**Comment by rchincha** [comment]
@tk-nguyen your PR inlines some of the documentation which I agree with since the upstream URL/docs may move/change.

**Review by rchincha — APPROVED** [review]
lgtm

**Pull request merged**